docs/guide/basics/sizing.md

```diff
@@ -0,0 +1,87 @@
+<script setup>
+import { Sandpack } from 'sandpack-vue3'
+import sandpackTheme from '../../.vitepress/theme/sandpackTheme.json'
+import sizeIconExample from './examples/size-icon-example/files.ts'
+import sizeIconCssExample from './examples/size-icon-css-example/files.ts'
+import sizeIconFontExample from './examples/size-icon-font-example/files.ts'
+import sizeIconTailwind from './examples/size-icon-tailwind-example/files.ts'
+</script>
+
+# Sizing
+
+By default, the size of all icons is `24px` times `24px`. The size is adjustable using the `size` prop and CSS.
```

Comment:
>`24px` times `24px`. 

I think 

>`24px` by `24px`.

Would sound better here

Reply:
Good one, 
Github tip: You can add suggestions with this little button in the review box.
<img width="32" alt="Screenshot 2023-09-24 at 21 06 44" src="https://github.com/lucide-icons/lucide/assets/11825403/5711443f-e7c1-4e89-9796-86fda0e91f06">